opencti-platform/opencti-front/src/utils/sessionStorage/setStackDataInSessionStorage/setStackDataInSessionStorage.ts

```diff
@@ -0,0 +1,17 @@
+type SetDataOnSessionStorageParams = {
+  valueToStore: unknown;
+  key: string;
+  stackValue: number;
+};
+const setStackDataInSessionStorage = ({ valueToStore, key, stackValue }: SetDataOnSessionStorageParams): void => {
+  const storedStackData = sessionStorage.getItem(key);
+  const currentStoredStackData = storedStackData ? JSON.parse(storedStackData) : [];
+
+  currentStoredStackData.unshift(valueToStore);
+
+  if (currentStoredStackData.length > stackValue) currentStoredStackData.pop();
```

Comment:
if we call this function with same key but different stackValue this would not work as intended.
We can slice the array instead of just pop I guess.